### DISCUSSION TITLE
Profile color

### DISCUSSION THREAD

**Discussion by Skylli202** [KinesisCorporation/Adv360-Pro-ZMK#352] [Q&A]
Hi,

I'd like to know if it is possible to configure RGB color of the profiles layers?

I've read that there is "RGB Underglow" in the ZMK docs, but I do not think it is what's used to set the layer color on my adv360 (ive the bluetooth version).

Thanks in advance for any help, very much appreciated

**Comment by ReFil** [comment]
Hi, Unfortunately this is not possible right now
